### PR TITLE
Extending the sandbox to include write capabilities

### PR DESCRIPTION
**Pull request by mzkrasner**
This update includes the following:

1. Extending the current sandbox to include the ability for users to perform writes
2. New module that authenticates users on the same node the sandbox uses, with limited scope only for the `CeramicDev` model
3. An integrated feed module that shows the users the most recent 8 records for `CeramicDev`
4. Instructions for the user to clone a repository to perform an additional write operation to close the loop

### CODE REVIEW

File: docs/composedb/sandbox.mdx [hunk 1]
```diff
@@ -67,6 +70,101 @@ Use the example queries below to access data related to the `Comments` model:
...
+</div>
+
+### Attest to Your Profile in a Local Environment
+
+Notice how some of the profiles have obtained attestations? We've intentionally omitted the ability to create an `AttestToDev` instance using this sandbox. You can attest to your own profile as well by cloning a local version of the application:
+
+```bash
+git clone https://github.com/ceramicstudio/sandbox-extension
+npm install
+npm run dev
```
Comment (JustinaPetr) [inline docs/composedb/sandbox.mdx:157]: Let's explain what will happen here - let's tell the users that after running this command the application will be up and running and they will be able to try out in their browser using the provided url.

File: docs/composedb/sandbox.mdx [hunk 2]
```diff
@@ -67,6 +70,101 @@ Use the example queries below to access data related to the `Comments` model:
...
+```
+
+Once you have the application running locally, you'll be able to sign in using your browser wallet. Be sure to use the same address and network you used when authenticating yourself on this page.
```
Comment (JustinaPetr) [inline docs/composedb/sandbox.mdx:160]: Let's specify that the address here stands for "wallet address".
Also, here, let's add a screenshot of what the user should see on the UI to self-authenticate. What do they have to press, etc?

File: docs/composedb/sandbox.mdx [hunk 3]
```diff
@@ -67,6 +70,101 @@ Use the example queries below to access data related to the `Comments` model:
...
+
+Once authenticated, you'll be able to verify your own profile as well as others.
```
Comment (JustinaPetr) [inline docs/composedb/sandbox.mdx:162]: Let's explicitly describe what should happen here. What button do users have to press, how will I know that I verified a profile. Let's add screenshots of how "success" looks like at this step.

File: docs/composedb/sandbox.mdx [hunk 4]
```diff
@@ -67,6 +70,101 @@ Use the example queries below to access data related to the `Comments` model:
...
+
+Finally, once you've verified your profile, return back to this page. You should now see your verification(s) counted for each unique verification you issued in your local environment.
```
Comment (JustinaPetr) [inline docs/composedb/sandbox.mdx:164]: Let's explicitly tell developers where do they have to look to see that - could be as simple as "check out the Developer Feed". Ideally, we would a link to this section.

**Review by JustinaPetr — CHANGES_REQUESTED** [review]

File: docs/composedb/sandbox.mdx [hunk 5]
```diff
@@ -67,6 +70,101 @@ Use the example queries below to access data related to the `Comments` model:
 <br />
 ---
 
+## Writing Data
+
+---
+
+#### Data Models
+
+The following sandbox section will allow you to create Ceramic documents using mutations. For this section, we will be using a different model definition:
```
[the remark this answers is not in the document]
Comment (mzkrasner) [reply docs/composedb/sandbox.mdx:79]: agreed! i've updated the use of "we"

File: docs/composedb/sandbox.mdx [hunk 6]
```diff
@@ -67,6 +70,101 @@ Use the example queries below to access data related to the `Comments` model:
...
+
+```GraphQL
```
[the remark this answers is not in the document]
Comment (mzkrasner) [reply docs/composedb/sandbox.mdx:81]: sounds good - I added "These have been included for your reference below and have already been deployed onto the node this sandbox uses."

File: docs/composedb/sandbox.mdx [hunk 7]
```diff
@@ -67,6 +70,101 @@ Use the example queries below to access data related to the `Comments` model:
...
+  profile: CeramicDev! @relationDocument(property: "attestedProfileId")
+  signal: Boolean!
+}
+```
+
+You will use this model to self-describe your own development skills, and will therefore require you to alter the given examples to meet your actual abilities.
+
+#### Authentication
+
+You will first have to self-authenticate on our Ceramic node. This is required for making mutation queries, whereas you do not have to be authenticated on a given node to read its data.
```
[the remark this answers is not in the document]
Comment (mzkrasner) [reply docs/composedb/sandbox.mdx:121]: definitely agree! i've added in this context

File: docs/composedb/sandbox.mdx [hunk 8]
```diff
@@ -67,6 +70,101 @@ Use the example queries below to access data related to the `Comments` model:
...
+
+<div style={{height: "100%", width: "100%", margin: "auto"}}><Connect /></div>
+
+Your browser wallet should create a prompt for you that looks something like this:
+
+<div style={{textAlign: 'center'}}>
+![Sign in prompt](/img/sign-in.png)
+</div>
+
+Notice how there's 1 model listed under "Resources" in the prompt - the model listed here is the StreamID of the model definition shown in GraphQL above that we've already deployed onto Ceramic. By signing this prompt, you are creating a temporary session that allows the current application to create edits on your behalf _only_ to the resources you've granted access to (in this case, our `CeramicDeveloper` schema).
```
[the remark this answers is not in the document]
Comment (mzkrasner) [reply docs/composedb/sandbox.mdx]: Just cleared up this confusion! Thank you

File: docs/composedb/sandbox.mdx [hunk 9]
```diff
@@ -67,6 +70,101 @@ Use the example queries below to access data related to the `Comments` model:
...
+
+### Creating a Record
+
+Now that you're authenticated, you'll be able to create a `CeramicDeveloper` profile of your own! Be sure to adjust the dummy queries below to meet your unique skill set.
```
[the remark this answers is not in the document]
Comment (mzkrasner) [reply docs/composedb/sandbox.mdx]: love it - added this as well

File: docs/composedb/sandbox.mdx [hunk 10]
```diff
@@ -67,6 +70,101 @@ Use the example queries below to access data related to the `Comments` model:
...
+
+</div>
+
+### Attest to Your Profile in a Local Environment
+
+Notice how some of the profiles have obtained attestations? We've intentionally omitted the ability to create an `AttestToDev` instance using this sandbox. You can attest to your own profile as well by cloning a local version of the application:
+
+```bash
+git clone https://github.com/ceramicstudio/sandbox-extension
+npm install
```
[the remark this answers is not in the document]
Comment (mzkrasner) [reply docs/composedb/sandbox.mdx:161]: added!

File: docs/composedb/sandbox.mdx (hunk 1 above)
Comment (mzkrasner) [reply docs/composedb/sandbox.mdx:162]: added!

File: docs/composedb/sandbox.mdx (hunk 2 above)
Comment (mzkrasner) [reply docs/composedb/sandbox.mdx]: added! great idea

File: docs/composedb/sandbox.mdx (hunk 3 above)
Comment (mzkrasner) [reply docs/composedb/sandbox.mdx]: cool - added!

File: docs/composedb/sandbox.mdx (hunk 4 above)
Comment (mzkrasner) [reply docs/composedb/sandbox.mdx]: great idea - added this too

File: docs/composedb/sandbox.mdx [hunk 11]
```diff
@@ -67,6 +70,101 @@ Use the example queries below to access data related to the `Comments` model:
...
+<div style={{height: "100%", width: "100%", margin: "auto", marginBottom: "5%", display : 'flex', flexDirection: "row"}}>
+
+
+<FeedExample  />
+
+</div>
+
+### Attest to Your Profile in a Local Environment
+
+Notice how some of the profiles have obtained attestations? We've intentionally omitted the ability to create an `AttestToDev` instance using this sandbox. You can attest to your own profile as well by cloning a local version of the application:
```
[the remark this answers is not in the document]
Comment (mzkrasner) [reply docs/composedb/sandbox.mdx]: almost forgot about this - just added!

**Review by JustinaPetr — APPROVED** [review]
Looks great, Mark! Let's merge it whenever we're ready.

**Pull request merged**